src/server/webSocket/handlePlayerSocket.js

```diff
@@ -14,10 +14,12 @@ export default async function (data) {
 
   switch (path) {
     case '/new': {
-      console.log('new piece nedded', data);
+      console.log('new piece needed', data);
 
       const { room } = data;
-      const newPlayer = new Player();
+      const socketId = io.id;
+
+      const newPlayer = new Player({ room, socketId });
```

Comment:
why new player ?
data should have a roomId + websocketId to have tracking info maybe (like which player asked for piece (help scoring), and which room(=game) to send new piece)